src/main/kotlin/com/example/fhirvalidator/controller/ValidateController.kt

```diff
@@ -49,11 +49,12 @@ class ValidateController(
     }
 
     fun validateResource(resource: IBaseResource): OperationOutcome? {
+        // KGM changed order so message defintion will override profiles added by capability statement
```

Comment:
You don't need your initials in this comment, Git will record that for you.

Reply:
The comment should explain why something is the way it is, rather than explaining what you changed (Git will do that aswell)

Reply:
Old EMIS habbit

Reply:
Those details are in the ticket